amo2kinto/tests/test_exporter.py

```diff
@@ -196,6 +196,71 @@ def test_addon_record_with_no_targetApplication_matching():
 """.decode('utf-8')
 
 
+def test_addon_record_group_by_blockID():
+    xml_tree = etree.Element(
+        'blocklist',
+        xmlns="http://www.mozilla.org/2006/addons-blocklist",
+        lastupdate='1459262434336'
+    )
+
+    data = ADDONS_DATA.copy()
+    data['versionRange'] = [{
+        "targetApplication": [
+            {"guid": "{ec8030f7-c20a-464f-9b0e-13a3a9e97384}",
```

Comment:
Since you use this guid twice, define it in a variable, it's gonna be easier to read the meaning of the test